nuxeo-core/nuxeo-core-api/src/main/java/org/nuxeo/ecm/core/api/security/ACP.java

```diff
@@ -42,6 +42,8 @@
  */
 public interface ACP extends Serializable, Cloneable {
 
+    String LEGACY_BEHAVIOR_PROPERTY = "nuxeo.security.acl.legacyBehavior";
```

Comment:
We shouldn't add this to the interface as it's not generally useful to users of the interface. Let's put it simply on `ACPImpl`.